### DISCUSSION TITLE
Is Keycloak affected by CVE-2024-1597?

### DISCUSSION THREAD

**Discussion by tnbreitkreutz** [keycloak/keycloak#27175] [Q&A] [answered]
CVE-2024-1597 recently showed up and being a 10/10. The JDBC version (42.7.1) currently utilized in the project would be affected, but I couldn't find any hint that the mentioned `PreferQueryMode=SIMPLE`. So, nothing to worry about?

**Comment by abstractj** [comment, marked as answer]
@tnbreitkreutz Thank you for bringing this to our attention. A new pull request has been submitted to address it: https://github.com/keycloak/keycloak/pull/27205. Regarding your question, we do not make use of `PreferQueryMode`, so we are not vulnerable, only affected.
